examples/connectivity/plot_adhd_covariance.py

```diff
@@ -95,13 +98,24 @@ def plot_matrices(cov, prec, title):
 
 # Displaying results ##########################################################
 print("-- Displaying results")
-title = ("Subject {0:d} GroupSparseCovariance "
-         "$\\alpha={1:.2e}$").format(plotted_subject, gsc.alpha_)
+atlas_imgs = image.iter_img(msdl_atlas_dataset.maps)
+atlas_region_coords = np.array(
```

Comment:
> I am afraid I don't see how a for loop is simpler.

It's a construct that more people know. For someone who doesn't know
Python, the list comprehension is a weird thing.